version.yaml

```diff
@@ -0,0 +1 @@
+logstash-output-dynatrace: '0.5.0'
```

Comment:
We could consider
```suggestion
logstash-output-dynatrace: '0.5.0.rc1'
```
as per https://guides.rubygems.org/patterns/#prerelease-gems.